src/main/java/com/jozufozu/flywheel/Flywheel.java

```diff
@@ -40,6 +46,8 @@ public class Flywheel {
 	public static final Logger LOGGER = LogUtils.getLogger();
 	private static ArtifactVersion version;
 
+	public static final Supplier<Boolean> IS_SODIUM_LOADED = Suppliers.memoize(() -> LoadingModList.get().getModFileById("sodium") != null);
```

Comment:
Confirm that this check will not pass if Embeddium decides to provide the Sodium mod ID.